.env.sample

```diff
@@ -349,6 +349,20 @@ JIBRI_BREWERY_MUC=jibribrewery
 # MUC connection timeout
 JIBRI_PENDING_TIMEOUT=90
 
+# If all clients have their audio and video muted and if Jibri does not
+# detect any data stream (audio or video) comming in, it will stop
+# recording after JIBRI_NO_MEDIA_TIMEOUT expires.
+JIBRI_NO_MEDIA_TIMEOUT="30 seconds"
+
+# If all clients have their audio and video muted, Jibri consideres this
+# as an empty call and stops the recording after JIBRI_ALL_MUTED_TIMEOUT expires.
+JIBRI_ALL_MUTED_TIMEOUT="10 minutes"
+
+# When detecting if a call is empty, Jibri takes into consideration for how
+# long the call has been empty already. If it has been empty for more than
+# DEFAULT_CALL_EMPTY_TIMEOUT, it will consider it empty and stop the recording.
+JIBRI_CALL_EMPTY_TIMEOUT="30 seconds"
```

Comment:
So this value overrides `DEFAULT_CALL_EMPTY_TIMEOUT` ?

Reply:
Oops, I copy/pasted the comments from [reference.conf](https://github.com/jitsi/jibri/blob/master/src/main/resources/reference.conf) file and forgot to rename the variable.

Reply:
fixed, thanks